code/modules/lighting/lighting_fake_sun_vr.dm

```diff
@@ -206,3 +206,14 @@ var/static/list/fake_sunlight_zs = list()
 			"color" = "#27024B"
 		)
 	)
+
+/obj/effect/fake_sun/underwater
+	do_weather = TRUE
+	weather_visuals_icon = 'icons/effects/weather.dmi'
+	weather_visuals_icon_state = "underwater"
```

Comment:
does the underwater fake sun actually work on the map? It does not seem to have any outdoor turfs in underwater z.

Reply:
Yeah, it works absolutely fine! All of the new underwater turfs are outdoors.

Reply:
FAKE SUN
FAKE SUN UNDER PARADISE